apps/admin-gui/src/app/shared/components/dialogs/add-edit-notification-dialog/tag-section/tag-section.component.html

```diff
@@ -3,13 +3,13 @@
     <button
       (click)="addTag('{' + tag[0] + '}')"
       matTooltip="{{'DIALOGS.NOTIFICATIONS_ADD_EDIT_MAIL.TAG_HINT' | translate }}"
-      class="tag-text w-25"
-      mat-stroked-button
+      class="tag-text w-5 mt-auto mb-auto me-2"
+      mat-icon-button
       matTooltipPosition="above">
-      {{tag[0]}}
+      <mat-icon>add_circle</mat-icon>
     </button>
-    <span class="w-75">
-      {{'DIALOGS.NOTIFICATIONS_ADD_EDIT_MAIL.' + tag[1] | translate}}
+    <span>
```

Comment:
please add align-items-center to the parent div (to vertically align the text with the icon)